### ISSUE TITLE
Should "Options.excluded" assignment be moved to the start of the lib?

### ISSUE THREAD

**Issue by kingo55** [mint-metrics/mojito-js-delivery#62]
Should this block be moved into the "Options" defaults definition at the start?

```js
    // General Mojito exclude feature
    // Mojito.options.excluded can be a boolean value or a function
    Options.excluded = !navigator.cookieEnabled;
```

https://github.com/kingo55/mojito-js-delivery/blob/9f313bd1ba59f1f8dcb80cc4df41b6b02b08f28c/lib/mojito.js#L1370-L1372

Into this definition:

```js
    var Options = {
        debug: false,
        // cookie duration (in days)
        cookieDuration: 60,
        // max length of error message
        maxErrorStackLength: 1000,
        // checking interval of utils.waitUntil
        waitInterval: 50,
        // default timeout of utils.waitUntil
        defaultWaitTimeout: 2000
    };
```

**Comment by allmywant** [comment]
You're right @kingo55 , we can do that.

**Comment by kingo55** [comment]
addressed by #77
